test/system/src/test/resources/systemTestConfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "resources": {
+    "requests": {
```

Comment:
there should be seperate resource requirements for controller, bk and segment store